### ISSUE TITLE
Add support IPFS

### ISSUE THREAD

**Issue by KonoromiHimaries** [claudiodangelis/qrcp#283]
I'm opening this issue because:

- [ ] I have found a bug
- [x] I want to request a feature
- [ ] I have a question
- [ ] Other

Add support ipfs to send via internet without own external ip and port unblocked

Can be work on public gate https://ipfs.github.io/public-gateway-checker/

**Comment by claudiodangelis** [comment]
Hello @KonoromiHimaries , thanks for suggesting this.
Unfortunately I am not too much familiar with IPFS and I would need some time to research first.
If you are a developer yourself and want to contribute, or you know someone who would, please feel free to step up.

Thanks again for this suggestion,
Claudio

**Comment by claudiodangelis** [comment]
Hello, this might be a bit out of scope.
I will close this issue for now, and I will take care of re-opening it in case something similar will implemented in the future.
Thanks!
Claudio
